.github/ISSUE_TEMPLATE/06_localization_data_issue.yml

```diff
@@ -0,0 +1,104 @@
+name: Localization Data Error
+description: Error in localization data.
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We're sorry you're experiencing issues with localization data.  There's some things you should be aware of when submitting the report that may help resolve your issue immediately, or will help us to troubleshoot the actual cause.
+        ## Quick Troubleshooting
+        To cover some commonly reported scenarios, there's some simple and quick troubleshooting that should be done.
+
+        Note that by default all localization and culture data comes from the host operating system.  The runtime SDK does not distribute its own set of data, and normally applications should not distribute their own copy either.  While there are occasionally errors when or how we retrieve this data, in most cases this means that any differences between what you expect and what you see are by design.
+        #### "Did nothing" (or only updated the computer) and localization has changed.
+        Culture data [is not stable](https://learn.microsoft.com/en-us/archive/blogs/shawnste/culture-data-shouldnt-be-considered-stable-except-for-invariant).  It is updated several times a year through host OS channels on all supported platforms.  This includes things like [spaces in time formats](https://github.com/dotnet/runtime/issues/95620).
+        #### Porting an application from .NET Framework or .NET Core < 5 to .NET Core 5+ on Windows.
+        Beginning with .NET Core 5, .NET Core on Windows now uses the [ICU (International Components for Unicode) libraries for localization](https://learn.microsoft.com/en-us/dotnet/core/extensions/globalization-icu).  While is is possible to [revert to the previous NLS datasource](https://learn.microsoft.com/en-us/dotnet/core/extensions/globalization-icu#use-nls-instead-of-icu), this is not recommended.
```

Comment:
> While is is possible to [revert to the previous NLS datasource]

I suggest to mention NLS which is used by .NET Framework. This may help the reader to under the difference between NETFX and .NET.